src/key.js

```diff
@@ -366,6 +366,22 @@ Key.prototype.getEncryptionKeyPacket = function() {
 };
 
 /**
+ * Encrypts all secret key and subkey packets
+ * @param  {String} passphrase
+ * @return {Boolean} true if all key and subkey packets encrypted successfully
+ */
+Key.prototype.encrypt = function(passphrase) {
+  if (this.isPrivate()) {
+    var keys = this.getAllKeyPackets();
+    for (var i = 0; i < keys.length; i++) {
+      keys[i].encrypt(passphrase);
+    }
+  } else {
+    throw new Error("Nothing to encrypt in a public key");
+  }
```

Comment:
ok for removing the else, but what about the indentation? could you please clarify?

i'm thinking to put simply:
```
  if (this.isPrivate()) {
    var keys = this.getAllKeyPackets();
    for (var i = 0; i < keys.length; i++) {
      keys[i].encrypt(passphrase);
    }
  }
```